packages/payload/src/errors/ValidationError.ts

```diff
@@ -28,6 +29,7 @@ export class ValidationError extends APIError<{
       errors: ValidationFieldError[]
       global?: string
       id?: number | string
+      req?: Partial<PayloadRequest>
```

Comment:
I'd add some JSDocs explaining that req needs to be passed through in order to resolve label functions that may be part of the `errors` array